react/react1/week1/todo-list-app/src/Components/TodoList.js

```diff
@@ -0,0 +1,39 @@
+import React from "react";
+import TodoItem from "./TodoItem";
+
+const todos = [
+  {
+    id: 1,
+    description: "Get out of bed",
+    deadline: "Wed Sep 13 2017",
+  },
+  {
+    id: 2,
+    description: "Brush teeth",
+    deadline: "Thu Sep 14 2017",
+  },
+  {
+    id: 3,
+    description: "Eat breakfast",
+    deadline: "Fri Sep 15 2017",
+  },
+];
+
+const TodoList = () => {
+  return (
+    <div className="todo-list">
+      <ul>
+        {todos.map((todo) => (
+          <TodoItem
+            key={todo.id}
+            description={todo.description}
```

Comment:
description and deadline are already contained in the todo parameter, so you don't really need these properties. :)

Reply:
yes, i missed it. Now fixed it. thanks for the review